docs/specs/Mountain State Adventure Destination/SPEC-09-adventure-hero-component/UNIFIED-ARCHITECTURE.md

```
@@ -0,0 +1,870 @@
+.cta-button:focus-visible {
+  outline: none;
+  ring: 2px solid var(--color-brand-orange);
+  ring-offset: 2px;
+  ring-offset-color: var(--color-brand-brown);
+}
+```
+
+### 5.5 Screen Reader Experience
+
+**Badge Announcement Pattern**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)